CMakeLists.txt

```diff
@@ -59,12 +59,17 @@ if (NOT (${CMAKE_VERSION} VERSION_LESS "3.18.0"))
   include(CheckLinkerFlag)
 endif()
 
+set(default_enable_threads ON)
+if(EMSCRIPTEN OR WASI)
```

Comment:
How about change this to?
+find_package(Threads QUIET)
+if (NOT (CMAKE_USE_PTHREADS_INIT OR CMAKE_USE_WIN32_THREADS_INIT) OR EMSCRIPTEN OR WASI)

Reply:
I have modified it. Please check commit [f21eefa](https://github.com/ivmai/bdwgc/commit/f21eefafc047240a4d971de5147bf64dd46725fe)